code/src/components/Details.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { useParams } from 'react-router-dom'
+import { Link } from 'react-router-dom'
+import styled from 'styled-components'
+
+
+const Backdrop = styled.section`
```

Comment:
We like that you used styled components and original size for this background image. Impressive!
We were wondering if this made it harder to style media queries?